src/olympia/search/indexers.py

```diff
@@ -58,19 +58,24 @@
 }
 
 
-def create_new_index(index_name=None, config=None):
+def create_new_index(index_name=None):
     """
     Create a new index for search-related documents in ES.
 
     Intended to be used by reindexation (and tests), generally a bad idea to
     call manually.
     """
-    if config is None:
-        config = {}
     if index_name is None:
         index_name = BaseSearchIndexer.get_index_alias()
-    config['settings'] = {'index': INDEX_SETTINGS}
```

Comment:
This meant the replica/shards settings were never used, since it was overriding them.